package.json

```diff
@@ -19,8 +19,10 @@
   "homepage": "https://github.com/bloglovin/lintlovin",
   "dependencies": {
     "dependency-check": "^2.1.0",
+    "grunt": "^0.4.5",
```

Comment:
As Grunt is added as a perrDependency to Lintlovin (and a devDependency) it shouldn't be here as well